docs/metrics.md

```diff
@@ -1,69 +1,112 @@
 # Fluo Metrics
 
-Fluo core is instrumented using [dropwizard metrics][1]. This allows fluo users to easily gather
-information about Fluo by configuring different reporters. While dropwizard can be configured to
-report Fluo metrics to many different tools, below are some tools that have been used with Fluo.
+A Fluo application can be configured (in [fluo.properties]) to report metrics. When metrics are 
+configured, Fluo will report will some 'default' metrics about an application that helps users 
+monitor its performance. Users can also write code to report 'application' metrics from their
```

Comment:
I would write :

Users can also write code to report application specific metrics.